server/apps/rules/routing_rule_validator.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import logging
+import re
+
+logger = logging.getLogger(__name__)
+
+
+class RoutingRuleValidator():
+
+    def is_valid_rule(self, item, rule_filter):
+        if not filter:
```

Comment:
You are right ```filter``` is always defined. It should be ```rule_filter```.  

@petrjasek  which exceptions are you referring?